src/frontends/tensorflow/README.md

```diff
@@ -0,0 +1,201 @@
+# OpenVINO TensorFlow Frontend
+
+The TensorFlow Frontend (TF FE) is a C++ based OpenVINO Frontend component that is responsible for reading and converting a TensorFlow model to an `ov::Model` object
+that further can be serialized into the Intermediate Representation (IR) format.
```

Comment:
Please also add "This is an internal API for OpenVINO that is used to implement user facing API like Model Optimizer, read_model function and OpenVINO TensorFlow integration. Regular users shouldn't use the frontend directly."

Reply:
Added. Thanks!